<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinRt::Internal::WinRtPackageDeploymentStepWidget</class>
 <widget class="QWidget" name="WinRt::Internal::WinRtPackageDeploymentStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>59</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Arguments:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leArguments"/>
   </item>
   <item>
    <widget class="QToolButton" name="btnRestoreDefaultArgs">
     <property name="text">
      <string>Restore Default Arguments</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
